zivid_camera/srv/LoadSettingsFromFile.srv

```diff
@@ -0,0 +1,2 @@
+string file_path
+---
```

Comment:
Would it not make sense to return a bool here indicating whether loading the settings file was successful or not?

Reply:
Answered below